docs/content/usage/push.en-us.md

```diff
@@ -47,7 +47,7 @@ git push -o repo.private=false -u origin main
 
 # Push To Create
 
-Push to create is a feature that allows you to push to a repository that does not exist yet in Gitea. This is useful for automation and for allowing users to create repositories without having to go through the web interface. This feature is disabled by default.
```

Comment:
> Push to create is a feature that allows you to push to a non-existing repository in Gitea and have the repository created automatically during the push.

Maybe the above? The purpose of the previous sentence is to help explain how/why this feature works.

Reply:
My bad... I skimmed middle of the sentence and understood something like "This feature does not exist yet in Gitea"